jupyterlab/staging/index.js

```diff
@@ -102,12 +102,12 @@ function main() {
   register.forEach(function(item) { lab.registerPluginModule(item); });
   lab.start({ ignorePlugins: ignorePlugins });
 
-  // Expose global lab instance when in dev mode or when toggled explicitly.
-  var exposeGlobal = (PageConfig.getOption('exposeGlobal') || '').toLowerCase() === 'true';
+  // Expose global app instance when in dev mode or when toggled explicitly.
```

Comment:
Files in staging should not be changed. They will be updated when a release happens. Can you revert your changes here?

Reply:
sure